### ISSUE TITLE
Columns are shown as rows 

### ISSUE THREAD

**Issue by troncomputers** [angular/flex-layout#1361]
# Bug Report

#### What is the expected behavior?
Original example
[https://flex-layout-material-seed.stackblitz.io](https://flex-layout-material-seed.stackblitz.io)

#### What is the current behavior?
Same example but with latest ver.
[https://angular-ivy-ouirmu.stackblitz.io](https://angular-ivy-ouirmu.stackblitz.io)
[EDITOR](https://stackblitz.com/edit/angular-ivy-ouirmu?file=src/app/app.component.html)

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Latest Angular 12.2.4, Windows 10, examples has TypeScript ver 4.0.2 but in my project is 4.2.3 with same behavior. Tested only on Edge (Chromium)

**Comment by pratapdev** [comment]
Did you found the root cause or any workaround.

**Comment by troncomputers** [comment]
Sadly no. I think Ivy has nothing to do with it. If you change it in my stackblitz it is still not working properly.

**Comment by pratapdev** [comment]
> Sadly no. I think Ivy has nothing to do with it. If you change it in my stackblitz it is still not working properly.

In your stackblitz there is no FlexLayoutModule import in app module

**Comment by mackelito** [comment]
Looking at https://stackblitz.com/edit/angular-ivy-ouirmu?file=src/app/app.module.ts seems to work just fine?

**Comment by troncomputers** [comment]
I'm not sure what happened, probably because I'm an idiot. The main reason why I opened this issue was a new project with latest versions of "everything". Tried to do columns -> wasn't working -> created stackblitz -> wasn't working either -> created issue here. I'm pretty sure I had imported FlexLayoutModule in my private project. Long story short... Created project again, from scratch and everything is working great 🙈🤦‍♂️🤷‍♂️ 

Should we close this issue? @pratapdev is it still not working for you?
